railties/lib/rails/generators/rails/scaffold_controller/USAGE

```diff
@@ -12,7 +12,7 @@ Description:
 Example:
     `rails generate scaffold_controller CreditCard`
 
-    Credit card controller with URLs like /credit_card/debit.
```

Comment:
Thought about that too but wouldn't listing seven routes be overkill?

Also the `scaffold` generator USAGE doesn't mention the URLs so I tried to be consistent with that.